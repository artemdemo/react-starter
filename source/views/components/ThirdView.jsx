import React from 'react';
import Icon from '../../components/Icon/Icon';

const ThirdView = () => {
    return (
        <>
            <Icon name='fire' />
            &nbsp;
            Third View
        </>
    );
};

export default ThirdView;
