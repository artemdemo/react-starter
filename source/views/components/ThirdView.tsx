import React from "react";
import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
import {faFire} from "@fortawesome/free-solid-svg-icons";

const ThirdView = () => {
    return (
        <>
            <FontAwesomeIcon icon={faFire} />
            &nbsp;
            Third View
        </>
    );
};

export default ThirdView;
