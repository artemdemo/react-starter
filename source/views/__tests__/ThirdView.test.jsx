import React from 'react';
import renderer from 'react-test-renderer';
import ThirdView from '../components/ThirdView';

describe('<ThirdView>', () => {
    it('Empty render', () => {
        const tree = renderer.create(
            <ThirdView />,
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
