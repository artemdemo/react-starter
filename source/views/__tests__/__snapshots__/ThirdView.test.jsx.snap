// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThirdView> Empty render 1`] = `
Array [
  <div
    data-mock="Icon"
  >
    {
  "name": "fire"
}
  </div>,
  "Â  Third View",
]
`;
